openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/TaskFeedCard/TaskFeedCardNew.component.tsx

```diff
@@ -230,7 +224,15 @@ const TaskFeedCard = ({
     assignee.type === 'team' ? checkIfUserPartOfTeam(assignee.id) : false
   );
   const hasEditAccess =
-    isAdminUser || isAssignee || (Boolean(isPartOfAssigneeTeam) && !isCreator);
+    (isAdminUser && !isTaskGlossaryApproval) ||
+    isAssignee ||
+    (Boolean(isPartOfAssigneeTeam) && !isCreator);
+
+  const isSuggestionEmpty =
+    (taskDetails?.suggestion === '[]' &&
+      taskDetails?.type === TaskType.RequestTag) ||
+    (!taskDetails?.suggestion &&
+      taskDetails?.type === TaskType.RequestDescription);
 
```

Comment:
I believe we can use `isEmtpty` method here from lodash. To check the values ?

Reply:
Done, have used isEqual check  from lodash ,since in the response of task for empty suggestions we are getting [] in form of string '[]'